libs/components/src/next/form/form-control/FormControl.ts

```diff
@@ -13,15 +13,27 @@ import { ERROR_MESSAGE_CUSTOM_TAG } from '../error-message/vl-error-message.comp
 import { BaseLitElement } from '@domg-wc/common-utilities';
 import 'reflect-metadata';
 
+export const FormControlDefaults = {
+    value: '',
+    name: '',
+    label: '',
+    block: false,
+    required: false,
+    disabled: false,
+    error: false,
+};
+
 export abstract class FormControl extends FormControlMixin(BaseLitElement) {
     id = '';
-    protected name = '';
-    protected label = '';
-    protected block = false;
-    protected required = false;
-    protected disabled = false;
-    protected error = false;
+    protected value = FormControlDefaults.value;
```

Comment:
Ik zou voorlopig de value niet in de FormControl plaatsen aangezien ik niet zeker ben dat dit gaat gelden voor elk component. Als we klaar zijn met alle nieuwe componenten en zien dat dat geldt voor ze allemaal kunnen we het refactoren.